CHANGELOG.md

```diff
@@ -33,20 +33,48 @@
  - #311 Resource Zone: update link to CapabilitiesObject
  - #329 Fix count on Computer System Collection
  - #339 Network Resource Block: add support for network sets
+ - #299 Include Storage resource blocks in a Resource Zone
+ - #309 Storage Resource Blocks in a Zone should be available only if storage controller is configured
+ - #270 Implement ComputerSystem for a composed resource
+ - #294 Include Network resource blocks in a Resource Zone
+ - #335 Reset action to a System is not working properly
+ - #317 Add support for Actions/Chassis.Reset in the Chassis for Server Hardware
+ - #271 Implement "@Redfish.CollectionCapabilities" in the Systems collection
+ - #337 Include information about Storage and Network for a Composed System
+ - #339 Network Resource Block: add support for network sets
+ - #341 Composed system: update NetworkInterfaces
+ - #349 Refactor Redfish.conf
+ - #263 Implement POST to create a composed resource
+ - #340 Composed System: Add ResourceBlocks in the Links
+ - #308 Update Links in the Storage Resource Block
+ - #350 Refactor odata.type
+ - #357 Set API version 500 for Event Service (workaround)
 
```

Comment:
I missing some issues (even that is before that two last sprints):
#264 
#273 
#276

Reply:
Updated.